src/test.js

```diff
@@ -125,7 +125,7 @@ Test.prototype = {
 		// ensure the callbacks are executed serially for each module
 		var callbackPromises = notStartedModules.reduce( ( promiseChain, startModule ) => {
 			return promiseChain.then( () => {
-				startModule.stats = { all: 0, bad: 0, started: now() };
```

Comment:
I believe these are different state, they are per-module ones. I think this would be unused currently (not incremented, not read). Can you double check?

Reply:
Awesome, I didn't catch that yet. I just checked locally and it seems to work as you suggested!